content/doc/book/blueocean/create-pipeline.adoc

```diff
@@ -0,0 +1,109 @@
+---
+layout: section
+title: Creating Pipelines in Blue Ocean
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Creating Pipelines in Blue Ocean
+
+Blue Ocean makes it easy to create new Pipelines or create Pipelines from an existing
```

Comment:
I intentionally tried not to use terms that differed from those used by Blue Ocean itself.  I agree the above is not great, but that is what Blue Ocean calls it.   I'm hesitant to have the docs stray from that. 

"importing from a `Jenkinsfile`" does not work - a Table is created in a Database using a SQL script, it not "imported" from a SQL file.  Also the Pipeline is reloaded and run, somewhat like a Python or Groovy script.